docs/_snippets/storybook-coverage-addon-optimized-config.md

```diff
@@ -3,11 +3,7 @@ export default {
   // Replace your-framework with the framework you are using (e.g., react-webpack5, vue3-vite)
   framework: '@storybook/your-framework',
   stories: ['../src/**/*.mdx', '../src/**/*.stories.@(js|jsx|mjs|ts|tsx)'],
-  addons: [
-    '@storybook/addon-essentials',
-    '@storybook/addon-interactions',
-    '@storybook/addon-coverage',
-  ],
+  addons: ['@storybook/addon-essentials', '@storybook/addon-coverage'],
```

Comment:
When you're able remove the references to `addon-essentials` from the snippets, not only this one, but the other ones related to this pr.